tests/fixtures/mock_api/app.py

```diff
@@ -195,9 +221,11 @@ async def ulc_get_files(file_id: str):
             current_upload_id="pending",
         )
 
-    raise HTTPException(
+    raise HttpException(
         status_code=404,
-        detail=(f'The file with the file_id "{file_id}" does not exist.'),
+        exception_id="fileNotRegistered",
+        description=f'The file with the file_id "{file_id}" does not exist.',
+        data={},
```

Comment:
Some of the exceptions ship data (such as https://github.com/ghga-de/epic-docs/blob/main/4-star-nosed_mole/api_definitions/rest/ucs.yaml#L332-L368), however, I don't see that the data field is used here anywhere. Could include that in the mocked response. Moreover, could you double-check whether you need that data for python error creation above in the `httpyexpect_translation.py`.